package.json

```diff
@@ -22,16 +22,16 @@
     "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
     "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
     "build-languages": "gulp languages",
-    "docker:up": "docker-compose -f docker/docker-compose.yml up",
+    "docker:up": "docker-compose -f docker/docker-compose.yml up -d",
```

Comment:
@withinboredom this was actually intended to run in the foreground. @enejb @roccotripaldi thought it fits their usage better. I'm impartial, but tending towards keeping the `-d`.

Reply:
I personally like keeping everything in one terminal. However, I just discovered that `yarn docker:up -d` works with the way it is in master allowing for both workflows instead of forcing `-d` on everyone.

Reply:
`-d` is mentioned in docs: https://github.com/Automattic/jetpack/blob/1f920e2c66d38f6e8d0f3eed4bc0544ad4374e30/docker/README.md#start-containers

Leaving it out indeed allows more flexibility and different workflows. :thumbsup:

Reply:
I removed `-d` for now as there seemed to be a consensus about it. We can chat more if anyone thinks it should be there by default.